src/main/java/org/jabref/gui/JabRefMain.java

```diff
@@ -25,12 +25,9 @@
 import org.jabref.migrations.PreferencesMigrations;
 import org.jabref.model.database.BibDatabaseMode;
 import org.jabref.preferences.JabRefPreferences;
-import org.jabref.rest.resources.RootResource;
+import org.jabref.rest.jabmap.JabMapHTTPServer;
```

Comment:
Please name the calss `JabMapHttpServer`. JabRef tries to follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case